src/components/RadioButtons/RadioButtonBox/RadioButtonBox.tsx

```diff
@@ -0,0 +1,165 @@
+import styled from '@emotion/styled';
+import React from 'react';
+import { pxToRem } from '../../../utils';
+import XSvg from '../../../assets/svg/icons/x-icon.svg';
+
+export type LabelPositions = 'top' | 'right' | 'bottom' | 'left';
+
+export interface RadioButtonBoxProps {
+  readonly defaultChecked?: boolean;
+  readonly color?: string;
+  readonly label?: string;
+  readonly labelPosition?: LabelPositions;
+  readonly gapSize?: number;
+  readonly width?: number;
+  readonly checkmarkSize?: number;
+  readonly borderWidth?: number;
+  readonly disabled?: boolean;
+  readonly required?: boolean;
+  readonly name?: string;
+  readonly value?: string | number;
+  readonly onChange?: () => void;
+}
+
+export const RadioButtonBox: React.FC<RadioButtonBoxProps> = ({
+  color = '#02c1b0',
+  label = 'Checkbox Label',
+  labelPosition = 'left',
+  gapSize = 10,
+  width = 30,
+  borderWidth = 2,
+  checkmarkSize = 65,
+  defaultChecked,
+  disabled,
+  onChange,
+  value,
+  name,
+  required,
+}) => {
+  const handleClick = () => {
+    onChange?.();
+  };
```

Comment:
This is an unnecessary wrapper at the moment.  We could just add this directly to the onClick

Reply:
Good catch. Updated

Reply:
Awesome work dude!